src/MicrosoftTeams.ts

```diff
@@ -861,6 +856,21 @@ export function showNotification(
   sendMessageRequest(parentWindow, "showNotification", params);
 }
 
+/**
+ * @private
+ * Hide from docs.
+ * ------
+ * execute deep link API.
+ * @param deepLink deep link.
+ */
+export function executeDeepLink(
+  deepLinkParameters: ExecuteDeepLinkParameters
```

Comment:
Do we really need an ExecuteDeepLinkParameters object? Couldn't we just have the developer pass the URL?

Reply:
will do that now.